lerna.json

```diff
@@ -0,0 +1,16 @@
+{
+  "lerna": "2.0.0-beta.30",
+  "version": "0.3.2",
+  "changelog": {
+    "repo": "apollostack/apollo-server",
+    "labels": {
```

Comment:
They are github issue/PR labels which @helfer needs to create then define there.
Later on it is used for creating changelog between versions